apis/workload/v1alpha1/types.go

```diff
@@ -114,6 +114,7 @@ type KubernetesApplicationStatus struct {
 
 // A KubernetesApplication defines an application deployed by Crossplane to a
 // Kubernetes cluster, i.e. a portable KubernetesCluster resource claim.
+// +kubebuilder:resource:categories={crossplane,workload}
```

Comment:
I think we want a `kubectl get workload` experience for OAM workloads, which would conflict with these ones. Would you mind leaving KubernetesApplication and friends out of the workload category until we sort that out?

Reply:
Sure thing! Meant to remove these, thanks for catching!